libweasyl/libweasyl/text.py

```diff
@@ -19,12 +19,12 @@ def slug_for(title):
 
 
 AUTOLINK_URL = (
-    r"(?P<url>(?isu)\b(?:https?://|www\d{,3}\.|[a-z0-9.-]+\.[a-z]{2,4}/)[^\s()"
+    r"(?P<url>\b(?:https?://|www\d{,3}\.|[a-z0-9.-]+\.[a-z]{2,4}/)[^\s()"
     r"<>\[\]\x02]+(?![^\s`!()\[\]{};:'\".,<>?\x02\xab\xbb\u201c\u201d\u2018"
     r"\u2019]))"
 )
 
-url_regexp = re.compile(AUTOLINK_URL)
+url_regexp = re.compile(AUTOLINK_URL, re.I | re.S)
```

Comment:
Yep.

> Note that for backward compatibility, the `re.U` flag still exists (as well as its synonym `re.UNICODE` and its embedded counterpart `(?u)`), but these are redundant in Python 3 since matches are Unicode by default for strings (and Unicode matching isn’t allowed for bytes).